### ISSUE TITLE
demo page

### ISSUE THREAD

**Issue by joaofgf** [goodybag/bootstrap-notify#33]
Hello,
If you use https, the demo page is not working. It uses a secure protocol (https) and all the include files are in non-secure protocol (http). Modern browsers does not include unsecure files on secure main file.
![2014-05-03_20-43-27](https://cloud.githubusercontent.com/assets/1038868/2871214/3f67f10e-d2fb-11e3-9053-5aa92b61df0a.jpg)

Solution 1: use protocol-relative urls:
instead of

```
http://getbootstrap.com/2.3.2/assets/js/jquery.js
```

use 

```
//getbootstrap.com/2.3.2/assets/js/jquery.js
```

Solution 2: use secure protocol
instead of

```
http://getbootstrap.com/2.3.2/assets/js/jquery.js
```

use 

```
https://getbootstrap.com/2.3.2/assets/js/jquery.js
```

Regards

**Comment by mlissner** [comment]
Yep. Hit this too. Anybody with HTTPS Everywhere installed will run into this.
